CHANGELOG.next.toml

```
@@ -11,6 +11,12 @@
 # meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client | server | all"}
 # author = "rcoh"
 
+[[smithy-rs]]
+message = "Fix tiny_map from_iter where one operation was being dropped"
```

```suggestion
message = "Fix bug in AWS JSON 1.x routers where, if a service had more than 14 operations, the router was created without the route for the 15th operation."
```

`TinyMap` is not public so users won't be able to know what the message refers to. This one is a bit more descriptive.